pbm/oplog/oplog.go

```diff
@@ -0,0 +1,25 @@
+package oplog
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+	"go.mongodb.org/mongo-driver/mongo"
+
+	"github.com/percona/percona-backup-mongodb/pbm"
+)
+
+// ListChunksBefore returns oplog chunks
+func ListChunksBefore(ctx context.Context, m *mongo.Client, ts primitive.Timestamp) ([]pbm.OplogChunk, error) {
```

Comment:
The rest of chunks extraction funcs are in pbm/pitr.go.

Reply:
I saw the `(*PBM).PITRGetChunksSlice()` method. But I need this function with this query

Reply:
I'm not saying ListChunksBefore is redundant but that similar methods/funcs now belong to different packages.

Reply:
This function is not a part of `PBM` or `Node` struct interface. To not lose it in a file somewhere in `pbm` module, I put it into this `oplog` module because we store oplog as chunks. For me, it is much more intuitive to keep such functions in `oplog` module than in `pbm`.